apps/prairielearn/elements/pl-file-editor/pl-file-editor.js

```diff
@@ -292,3 +283,35 @@ window.PLFileEditor.prototype.b64EncodeUnicode = function (str) {
     }),
   );
 };
+
+window.PLFileEditor.prototype.preview = {
+  html: (value) => value,
+  markdown: (() => {
+    let markdownRenderer = new showdown.Converter({
+      literalMidWordUnderscores: true,
+      literalMidWordAsterisks: true,
+    });
+
+    return async (value) => markdownRenderer.makeHtml(value);
+  })(),
+  dot: (() => {
+    let vizPromise = null;
+    return async (value) => {
+      try {
+        // Only load/create instance on first call.
+        if (vizPromise == null) {
+          vizPromise = (async () => {
+            // eslint-disable-next-line import/no-unresolved
+            await import('@viz-js');
+            return await Viz.instance();
```

Comment:
Nit: there's also an ESM entrypoint. If you changed the import map to point to `@viz-js/viz/lib/viz-standalone.mjs` instead, you could avoid the need to use global values:

```suggestion
            const { instance } = await import('@viz-js');
            return await instance();
```

Reply:
Done in 05f543f2cdb4fce0802acba017a5a09de2f2936e.